SlickEngagement_Plugin.php

```diff
@@ -516,37 +523,33 @@ private function echoPageBootData()
         }
 
         global $wp;
-        
-        $transient_name = $this->getTransientName(); //Name for WP Transient Cache API Key
+
+        $transient_name = $this->getTransientName(); // Name for WP Transient Cache API Key
+        $transient_name_ttl = $this->getTransientName() . '_ttl'; // Name for WP Transient Cache TTL Key
```

Comment:
We can improve this a bit, since function calling requires more time compared to variable. it saves micro seconds but it can add up when we have 30mills PV a month.

This is what I have in mind:
```
 $transient_name_ttl = $transient_name . '_ttl';
```